### DISCUSSION TITLE
咨询关于 client loader 的行为

### DISCUSSION THREAD

**Discussion by Lzzzzzq** [umijs/umi#11560] [Q&A]
### 背景
  
在使用 umi 4 中的 client loader 能力时，直接通过 url 访问单独页面，可以正常发起 client loader 中的请求
  
但如果当前页面（父页面）下有嵌套子页面的话，在子页面间路由切换时，也会导致父页面的 client loader 重新发起

在使用 remix 框架的时候，似乎没有遇到这个问题

### 最小复现
  
https://codesandbox.io/p/sandbox/sweet-murdock-cp95hh?file=/src/pages/index.tsx:17,7

**Comment by fz6m** [comment]
client loader 属于实验性功能，如有重复请求问题，可以自己根据路径建立缓存，如对 client loader 有特别喜好，建议使用 remix 或原生 react 应用搭配 react-router 的 `loader` 。
